src/__tests__/posthog-core.test.ts

```diff
@@ -52,21 +84,108 @@ describe('posthog core', () => {
 
                 console.error = jest.fn()
                 const { posthog, onCapture } = setup()
-
                 for (let i = 0; i < 100; i++) {
-                    posthog.capture(eventName, properties)
+                    posthog.capture(eventName, eventProperties)
                 }
                 expect(onCapture).toHaveBeenCalledTimes(100)
                 onCapture.mockClear()
                 ;(console.error as any).mockClear()
-                posthog.capture(eventName, properties)
-                expect(onCapture.mock.calls).toEqual([])
-                expect(console.error).toHaveBeenCalledTimes(1)
+                for (let i = 0; i < 50; i++) {
+                    posthog.capture(eventName, eventProperties)
+                }
+                expect(onCapture).toHaveBeenCalledTimes(1)
+                expect(onCapture.mock.calls[0][0]).toBe('$$client_ingestion_warning')
```

Comment:
This test was broken before, it relied on state from the previous test and failed when run in isolation.